declarations/plotly.js/index.d.ts

```diff
@@ -3,8 +3,11 @@ import 'plotly.js'
 declare module 'plotly.js' {
   type Update = { x: Array<number[]>, y: Array<number[]> }
   type ShortLayout = { width: number, height: number }
+  type ShortMarker = {size: number}
+  type ShortData = {x: Array<number[]>, y: Array<number[]>, z: Array<number[]>, mode: string, type: string, marker: ShortMarker, name: string}
```

Comment:
Oh does that work now? I used to get errors using the short-hand way for multi-dimensional arrays at my previous job. Wonder if that was an issue with the old school typings or the compiler.